devices/common_patches/mpu_unprefix_registers.yaml

```diff
@@ -0,0 +1,220 @@
+ADC:
+  _strip:
+    - "ADC_"
```

Comment:
As there is only single entry in the list you could write it shorter
```suggestion
ADC:
  _strip: ADC_
```

Reply:
That looks much better. Thanks!